src/diffusers/utils/import_utils.py

```diff
@@ -369,6 +369,44 @@ def is_torchsde_available():
     return _torchsde_available
 
 
+def _is_torch_fp16_available(device):
+    if not is_torch_available():
+        return False
+
+    import torch
```

Comment:
Importing torch would affect the lazy import speed. I think we can move these functions into `testing_utils` and remove importing them in `utils`.